HyperAlloc/pom.xml

```diff
@@ -397,13 +397,13 @@
                     <archive>
                         <manifest>
                             <addClasspath>false</addClasspath>
-                            <mainClass>com.amazon.corretto.benchmark.heapothesys.Heapothesys</mainClass>
+                            <mainClass>com.amazon.corretto.benchmark.hyperalloc.HyperAlloc</mainClass>
```

Comment:
I think the package should keep heapothesys as the parent package, but HyperAlloc as the child.  So  something like: com.amazon.corretto.benchmark.heapothesys.HyperAlloc

Reply:
What is implemented in the current draft pull request is that the original heapothesys root has been moved into a HyperAlloc subdirectory and the heapothesys "application" has been renamed HyperAlloc.  It source is contained within the java.com.amazon.corretto.benchmark.hyperalloc package, all of which is bundled within the HyperAlloc subdirectory.